src/components/UserProfile/UserProfile.jsx

```diff
@@ -631,17 +637,16 @@ function UserProfile(props) {
                 className="fa fa-info-circle"
                 onClick={toggleInfoModal}
               />{' '}
-              {canEdit && (
-                <>
-                  <ActiveCell
-                    isActive={userProfile.isActive}
-                    user={userProfile}
-                    onClick={() => {
-                      setActiveInactivePopupOpen(true);
-                    }}
-                  />
-                </>
-              )}
+              <>
+                <ActiveCell
+                  isActive={userProfile.isActive}
+                  user={userProfile}
+                  canChange={canChangeUserStatus}
+                  onClick={() => {
+                    setActiveInactivePopupOpen(true);
+                  }}
+                />
+              </>
```

Comment:
```suggestion
              <ActiveCell
                isActive={userProfile.isActive}
                user={userProfile}
                canChange={canChangeUserStatus}
                onClick={() => {
                  setActiveInactivePopupOpen(true);
                }}
              />
```
The fragment is not necessary.